mod-linked-data/src/main/resources/citation/mod-linked-data/create-ref-data.feature

```diff
@@ -11,7 +11,7 @@ Feature: Create refrerence data
     And request
       """
       {
-        "id": "3be866c2-33c4-4f8a-a0e7-654c674e8854",
+        "id": "526aa04d-9289-4511-8866-349299592c18",
```

Comment:
The "id" that we use was not in sync with id in reference data. This caused some caching issue and resulted in test failure.